src/query_parser/v2/parser.y

```diff
@@ -1195,21 +1195,23 @@ term(A) ::= TERM(B) . {
 
 term(A) ::= NUMBER(B) . {
   A = B;
+  A.type = QT_NUMERIC;
 }
 
 term(A) ::= SIZE(B). {
   A = B;
+  A.type = QT_SIZE;
 }
```

Comment:
When do we get here? I don't see the lexer sending a `SIZE` anywhere.

Reply:
We get there in queries like this:
`FT.SEARCH idx '23' dialect 2`
The lexer send it:
https://github.com/RediSearch/RediSearch/blob/bda189a9dfca9f51c88284eb3939796710f28315/src/query_parser/v2/lexer.rl#L151

Reply:
Oh right, missed that somehow.